src/task 22/task22.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <style>
+            table{
+                border-collapse: collapse;
+            }
+            td,th {
+                padding: 3px 8px;
+                border: 1px solid black;
+            }
+            th{
+                text-align: left;
+            }
+        </style>
+    </head>
+    <body>
+        <div id = "table"></div>
+        <script>
+            var Table = document.querySelector("#table");
+            var MOUNTAINS = [
+                {name: "Kilimanjaro", height: 5895, country: "Tanzania"},
+                {name: "Everest", height: 8848, country: "Nepal"},
+                {name: "Mount Fuji", height: 3776, country: "Japan"},
+                {name: "Mont Blanc", height: 4808, country: "Italy/France"},
+                {name: "Vaalserberg", height: 323, country: "Netherlands"},
+                {name: "Denali", height: 6168, country: "United States"},
+                {name: "Popocatepetl", height: 5465, country: "Mexico"}
+            ];
+            var headers = Object.keys(MOUNTAINS[0]);
```

Comment:
What if first element contains not a full list of properties? Think about a way to resolve it in a more generic way.